src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
@davidangb ok, so it sounds like the bulk of the changes for this ticket should actually occur in rawls and not orch. rawls has no idea what it means to sync a whitelist so it would need to learn.